google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/schema/RunBatchDdl.java

```diff
@@ -31,8 +33,23 @@
 
   private final Action schemaAction;
 
+  private final List<String> statements;
+
+  /**
+   * Constructs the {@link RunBatchDdl} auxiliary database object.
+   * @param schemaAction the DDL mode being used for schema generation.
+   */
   public RunBatchDdl(Action schemaAction) {
     this.schemaAction = schemaAction;
+    this.statements = new ArrayList<>();
+    this.statements.add("RUN BATCH");
+  }
+
+  /**
+   * Add a statement to run after the DDL batch is completed.
```

Comment:
I'd add a comment for why this may be necessary.

Reply:
Done.